Resources/Textures/Clothing/tailcover.rsi/meta.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-4.0",
+    "copyright": "TailCover by gentlebutter (588410099190726656) and ghostprince (115894681304301569)",
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "states": [
+        {
+            "name": "tailcover-vulpkanin",
+            "directions": 4
+        },
+        {
+            "name": "tailcover-reptilian",
+            "directions": 4
+        },
+        {
+            "name": "tailcover-felinds",
+            "directions": 4
+        }
+    ]
+}
```

Comment:
If you check in Resources/Textures/DeltaV/Mobs/Customization/Harpy, there is a stray png file listed as harpy_wingcover.png not present in any RSI. Do you mind moving it to this folder, and rename it to tailcover-harpy, then add the TailVisual component and required visual layers to Harpy?

Reply:
Will do.

Reply:
Thanks. I'm having Still-Icarus make a wing cover for Harpies too, after this is merged I'll implement a wing cover system for Harpies so that they can finally have actual proper hardsuits. 
![Fari_heart](https://github.com/Simple-Station/Einstein-Engines/assets/16548818/a5e892cf-765a-4e36-95f7-d53bfaa07d8b)